packages/atomic/create-atomic/src/plopfile.ts

```diff
@@ -38,6 +38,20 @@ export default async function (plop: NodePlopAPI) {
     return platformClientInstance;
   }
 
+  async function createImpersonateApiKey(
```

Comment:
If we move that step at this early stage, what will the user get if he does not have the privileges to create API keys?